recipes/podofo/all/patches/0001-fix-cmake-0.9.6.patch

```diff
@@ -0,0 +1,164 @@
+--- a/CMakeLists.txt
```

Comment:
This patch is heavy.
It would be nice if, or at least some portions of it, could be upstreamed.

Reply:
I agree. `fix cmake` doesn't explain much on what exactly is fixed and why...
in general, if we generate compatible find modules and config files, it shouldn't make any difference (in theory)...
do we fail to set up some targets or variables properly? do we need fixes in conan generator?
I believe, our goal should be a transparent integration here
/cc @uilianries

Reply:
This patch is required for several reasons:
- to use our generated Find module files:
  - upstream prepends its own cmake dirs in CMAKE_MODULE_PATH, with its fragile Find module files
  - usptream doesn't use official Find module names, nor CMake imported targets (very old CMake style: CMake 2.6 !), it's the main reason why the patch is so "big".
- ensure reproducible build: upstream tries to find dependencies, and if not found disable a feature. So, without patch adding these options, we can't avoid to provide all dependencies, because if not it could find system dependencies.

I can add a comment in patches.

Reply:
And yes I agree that we should submit a PR upstream, but it takes time.

Reply:
We should open a (sticky) issue where we collect recipes that contain patches that are candidate for upstream.